src/main/java/com/direwolf20/mininggadgets/client/particles/laserparticle/LaserParticle.java

```diff
@@ -132,6 +133,8 @@ public void tick() {
         Vec3 playerPos = player.position().add(0, player.getEyeHeight(), 0);
         Vec3 blockPos = new Vec3(sourceX, sourceY, sourceZ);
         Vec3 look = player.getLookAngle(); // or getLook(partialTicks)
+
+
         //The next 3 variables are directions on the screen relative to the players look direction. So right = to the right of the player, regardless of facing direction.
```

Comment:
Nitpick: Can this only have 1 enter, not 2?

Reply:
Whoops, missed that from moving some lines around, thanks for the find